sda-commons-server-trace/src/main/java/org/sdase/commons/server/trace/filter/TraceTokenServerFilter.java

```diff
@@ -63,6 +63,7 @@ private Optional<String> extractTokenFromRequest(ContainerRequestContext request
   private void addTokenToMdc(String token) {
     if (MDC.getMDCAdapter() != null) {
       MDC.put(RequestTracing.TOKEN_MDC_KEY, token);
+      MDC.put("TraceID", token);
```

Comment:
I would add inline comments here to make clear what Trace Token/ID is from which context. Could we name it `OtelTraceId` or is the name important for matching in Grafana?